turbopack/crates/turbopack-ecmascript/tests/analyzer/graph/for-infinite-break/input.js

```diff
@@ -0,0 +1,14 @@
+// app/page.js
+import { assert } from './assert'
+
+function test(src) {
+  for (;;) {
```

Comment:
What is about `while(true) { ... }` and `do { ... } while(true)`?

Reply:
Fixed.
However, all of these `end_early_return_block` calls in `if`s/`try`s/block statements are actually incorrect.
We need a `end_early_break_block` (for `breaks`), for `if`s/`try`s/block statements
And then `end_early_break_block` should only be called for functions (and the whole module)

```
function B() {
  while (true) {
	// currently completely ignored, should early return the loop
    break 

    // should early return the function, and it does
	// but this is never reached, it will make Turbopack ignore the rest of the function
    return 
  }
  // ignored due to the `return`, though the `break` before that actually prevents the `return`
  calledB1()
}
```

Reply:
What this PR does is make `return` actually behave like `break`. Then the example above works correctly